metron-deployment/packaging/docker/deb-docker/pom.xml

```diff
@@ -0,0 +1,253 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>metron-debs</artifactId>
+    <packaging>pom</packaging>
+    <name>metron-debs</name>
+    <version>0.4.2</version>
+    <parent>
+        <groupId>org.apache.metron</groupId>
+        <artifactId>metron-deployment</artifactId>
+        <version>0.4.2</version>
+        <relativePath>../../..</relativePath>
+    </parent>
+    <description>DEB Package Builder for Apache Metron</description>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <docker.tag>deb-docker</docker.tag>
+        <metron_dir>${project.parent.relativePath}/..</metron_dir>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.0.0</version>
+                <configuration>
+                    <filesets>
+                        <fileset>
+                            <directory>target</directory>
+                            <followSymlinks>false</followSymlinks>
+                            <includes>
+                                <include>**/*</include>
+                            </includes>
+                        </fileset>
+                    </filesets>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.0.1</version>
+                <executions>
+                    <execution>
+                        <id>copy-deb-sources</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/target</outputDirectory>
```

Comment:
This is the same as what is in the docker rpm build, is there a way that they can share this and it doesn't have to be separately maintained?

Reply:
Yes it is the same.  I had the same thought, but could not think of a way to do it.  If you or anyone has suggestions, I'd be open to improving that.

Reply:
All I can think of is refactoring so that they both build out of the same directories ( since i don't think deb cares like rpm does ).  Have the copy phase a level up or something.  that get's complicated quickly too.

Reply:
I bring it up because my 777 had a lot of spec and copy-resource work, so remember to have do it in both places hits home ;)